.github/labeler.yml

```diff
@@ -0,0 +1,38 @@
+# Configuration for GitHub Action `.github/workflows/pull-request-labeler.yml`.
+# See https://github.com/marketplace/actions/labeler for more information.
+
+# Only documentation changes
+docs:
+  - all: [ '**/*.md' ]
+
+# Only test changes
+dev/testing:
+  - all: [ '{spec/**,integration/**}' ]
+
+# Changes to Profiling
+profiling:
+  - any: [ 'lib/datadog/profiling/**' ]
+
+# Changes to CI-App
+ci-app:
+  - any: [ 'lib/datadog/ci/**' ]
+
+# Changes to ASM
+appsec:
+  - any: [ 'lib/datadog/appsec/**' ]
+
+# Changes to Tracing integrations
+integrations:
+  - any: [ 'lib/datadog/tracing/contrib/**' ]
+
+# Only repository GitHub changes
+dev/github:
+  - all: [ '.github/**' ]
+
+# Only repository CI changes
+dev/ci:
+  - all: [ '{.circleci/**,.gitlab-ci.yml}' ]
+
+# Version bump pull request
+release:
+  - all: [ '{CHANGELOG.md,lib/ddtrace/version.rb,gemfiles/**}' ]
```

Comment:
Really minor: Since we've done a few releases where we didn't bump all the gemfiles, I wonder if it would make sense to relax this rule to take that into account.

Reply:
Is that so? When a release is done at least `ddtrace (x.y.z)` should get an update in the lockfiles, correct?

Reply:
Yes and no. The way we've set up CI is that the CI runners will do the update locally anyway, thus we can actually not update it and things will still be fine.

But we do want the lockfiles to be updated from time to time, so this is why we have it as a step of the release process.

So my question/suggestion was more of a: should we consider here the exceptional cases where we do a release and skip the lockfile update?

Reply:
I'm removing gemfile bump from this list, to ensure we have no false positives on the `release` tag.